7_drivers_munge.yml

```diff
@@ -16,6 +16,7 @@ targets:
   7_drivers_munge:
     depends:
       - "7_drivers_munge/out/7_drivers_munge_tasks.ind"
+      - "7_drivers_munge/out/7_all_local_drivers.rds.ind"
 
   meteo_file_dir:
     command: c(I("7_drivers_munge/out"))
```

Comment:
When building `7_drivers_munge`, I got stopped at `cell_group_files`:
```
> scmake('cell_group_files')
Starting build at 2020-03-29 11:00:40
<  MAKE > cell_group_files
[    OK ] replete_time_range
[    OK ] cell_ind_dir
[    OK ] NLDAS_grid
[    OK ] feature_NLDAS_cells
[    OK ] nldas_cells
[ BUILD ] cell_group_files                                                  |  cell_group_files <- calc_cell_group...
 Error in sc_indicate("", data_file = filename) : 
  data_file must exist if specified 
```

Reply:
The error is occurring because it's attempting to hash non-existent files:
```
  variable                                                         filename
1 dlwrfsfc 6_drivers/log/feather/cellgroup_time[0.359420]_var[dlwrfsfc].yml
2  pressfc  6_drivers/log/feather/cellgroup_time[0.359420]_var[pressfc].yml
3  apcpsfc  6_drivers/log/feather/cellgroup_time[0.359420]_var[apcpsfc].yml
4  vgrd10m  6_drivers/log/feather/cellgroup_time[0.359420]_var[vgrd10m].yml
5  ugrd10m  6_drivers/log/feather/cellgroup_time[0.359420]_var[ugrd10m].yml
6 dswrfsfc 6_drivers/log/feather/cellgroup_time[0.359420]_var[dswrfsfc].yml
7   spfh2m   6_drivers/log/feather/cellgroup_time[0.359420]_var[spfh2m].yml
8    tmp2m    6_drivers/log/feather/cellgroup_time[0.359420]_var[tmp2m].yml
```
which I indeed do not have on my computer. However, I do have all these .ind files:
```
> dir("6_drivers/log/feather")
 [1] "6_fetch_append_cell_tasks.ind"                  "6_fetch_new_cell_tasks.ind"                    
 [3] "cellgroup_time[0.346848]_var[apcpsfc].yml.ind"  "cellgroup_time[0.346848]_var[dlwrfsfc].yml.ind"
 [5] "cellgroup_time[0.346848]_var[dswrfsfc].yml.ind" "cellgroup_time[0.346848]_var[pressfc].yml.ind" 
 [7] "cellgroup_time[0.346848]_var[spfh2m].yml.ind"   "cellgroup_time[0.346848]_var[tmp2m].yml.ind"   
 [9] "cellgroup_time[0.346848]_var[ugrd10m].yml.ind"  "cellgroup_time[0.346848]_var[vgrd10m].yml.ind" 
[11] "cellgroup_time[0.351500]_var[apcpsfc].yml.ind"  "cellgroup_time[0.351500]_var[dlwrfsfc].yml.ind"
[13] "cellgroup_time[0.351500]_var[dswrfsfc].yml.ind" "cellgroup_time[0.351500]_var[pressfc].yml.ind" 
[15] "cellgroup_time[0.351500]_var[spfh2m].yml.ind"   "cellgroup_time[0.351500]_var[tmp2m].yml.ind"   
[17] "cellgroup_time[0.351500]_var[ugrd10m].yml.ind"  "cellgroup_time[0.351500]_var[vgrd10m].yml.ind" 
[19] "cellgroup_time[0.359420]_var[apcpsfc].yml.ind"  "cellgroup_time[0.359420]_var[dlwrfsfc].yml.ind"
[21] "cellgroup_time[0.359420]_var[dswrfsfc].yml.ind" "cellgroup_time[0.359420]_var[pressfc].yml.ind" 
[23] "cellgroup_time[0.359420]_var[spfh2m].yml.ind"   "cellgroup_time[0.359420]_var[tmp2m].yml.ind"   
[25] "cellgroup_time[0.359420]_var[ugrd10m].yml.ind"  "cellgroup_time[0.359420]_var[vgrd10m].yml.ind" 
```

Reply:
Would it work to either (1) hash the .yml.ind files rather than the .yml files or (2) git commit the .yml files? I think either of those solutions would make the hashing effective, but I'm less sure what effect they would have on downstream uses of `cell_group_files`.

Reply:
Good ideas. I will try these out when I return to my comp and push up changes. 

Yes, that object target is going to fail 🤦‍♂ given those files aren't local. Whoops.